tests/functional/conftest.py

```diff
@@ -26,7 +26,7 @@ def pytest_sessionstart(session: pytest.Session) -> None:
 
     try:
         result = subprocess.run(
-            shlex.split(f"pdm build -v --no-sdist --dest {deptry_wheel_path}", posix=sys.platform != "win32"),
+            shlex.split(f"uvx pdm build -v --no-sdist --dest {deptry_wheel_path}", posix=sys.platform != "win32"),
```

Comment:
We could use what's in https://docs.astral.sh/uv/guides/publish/#building-your-package, but there are other things I'd like to rework in functional tests, so keeping `pdm` for building for the moment.